crates/holochain_types/src/app/app_manifest/app_manifest_v0.rs

```diff
@@ -140,6 +143,10 @@ pub enum CellProvisioning {
     /// Always create a new Cell when installing this App
     Create { deferred: bool },
 
+    #[deprecated(
+        since = "0.6.0-dev.17",
+        note = "for late binding, bundle your own coordinators and for calling cells of other apps, use bridge calls"
```

Comment:
```suggestion
        note = "For late binding, update the coordinators of a DNA. For calling cells of other apps, bridge calls can be used."
```

Reply:
Or something like this, to be in agreement with what's written in the changelog.